### PR TITLE
fix(style): update the buttons of the example tabs to be displayed as links

### PR DESCRIPTION
**Pull request by dhenneke**
### Description

https://github.com/swagger-api/swagger-ui/pull/7464 changed the tabs in the example section from links to buttons to improve the accessibility. This PR changes the styles of the buttons so they look like the old links:

#### Before

image

#### After

image

### My PR contains... 

- [ ] No code changes (`src/` is unmodified: changes to documentation, CI, metadata, etc.)
- [ ] Dependency changes (any modification to dependencies in `package.json`)
- [ ] Bug fixes (non-breaking change which fixes an issue)
- [x] Improvements (misc. changes to existing features)
- [ ] Features (non-breaking change which adds functionality)

### My changes...
- [ ] are breaking changes to a public API (config options, System API, major UI change, etc).
- [ ] are breaking changes to a private API (Redux, component props, utility functions, etc.).
- [ ] are breaking changes to a developer API (npm script behavior changes, new dev system dependencies, etc).
- [x] are not breaking changes.

### Documentation
- [x] My changes do not require a change to the project documentation.
- [ ] My changes require a change to the project documentation.
- [ ] If yes to above: I have updated the documentation accordingly.

### Automated tests
- [x] My changes can not or do not need to be tested.
- [ ] My changes can and should be tested by unit and/or integration tests.
- [ ] If yes to above: I have added tests to cover my changes.
- [ ] If yes to above: I have taken care to cover edge cases in my tests.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by char0n** [comment]
please build

**Comment by char0n** [comment]
@dhenneke Thanks!

I've tested this manually before it went to master, but unfortunately I didn't use "Incognito Window" to eliminate all cached files. Glad we have you to spot it and fix it in no time ;]

**Comment by dhenneke** [comment]
I assume that the color is not propagated to the button: 
image

**Comment by dhenneke** [comment]
Yep. When I add the color it looks better:

### Without color
image

### With color
image

File: src/style/_layout.scss [hunk 1]
```diff
@@ -500,6 +500,17 @@
         {
             font-weight: bold;
         }
+
+        button
```
Comment (char0n) [inline src/style/_layout.scss:504]: Can I suggest one additional change?
```suggestion
        button.tablinks
```

**Review by char0n — APPROVED** [review]
LGTM. Thanks for contributing! Have a look at my last comment regarding making the button selector more specific

File: src/style/_layout.scss (hunk 1 above)
Comment (dhenneke) [reply src/style/_layout.scss]: Done 👍

**Comment by char0n** [comment]
Tested in incognito window now to verify regression is gone ;]

**Comment by char0n** [comment]
Thanks again!

**Pull request merged**